src/BloomExe/Book/HtmlDom.cs

```diff
@@ -886,7 +886,7 @@ public static string GetAttributeValue(XmlElement elt, string name)
 			return attr.Value;
 		}
 
-		public static void FindFontsUsedInCss(string cssContent, HashSet<string> result)
+		public static void FindFontsUsedInCss(string cssContent, HashSet<string> result, bool firstFontInEachList)
```

Comment:
This parameter wants an explanation for some future dev. Why would we only want the first font? Why don't we want it when we make epubs?